examples/eepromfstest/Makefile

```diff
@@ -1,35 +1,26 @@
-PROG_NAME = eepromfstest
+BUILD_DIR = build
+include $(N64_INST)/include/n64.mk
 
-ROOTDIR = $(N64_INST)
-GCCN64PREFIX = $(ROOTDIR)/bin/mips64-elf-
+.DEFAULT_GOAL := all
```

Comment:
I would say this is superfluous. It's also not very common (I for one didn't know that it existed), so I would left it out from examples that should show how to compactly achieve a build.